likecoin/js/admin-settings/src/store/web-monetization-store.js

```diff
@@ -37,7 +37,7 @@ const actions = {
 };
 
 const selectors = {
-  selectPaymentPointer: (state) => state.pointer || '',
+  selectPaymentPointer: (state) => state,
```

Comment:
What does this change do?

Reply:
The state is now called DBPaymentPointer, so state.pointer will become undefined.